include/psen_scan/scanner.h

```diff
@@ -51,6 +56,13 @@ class Scanner : public vScanner
   LaserScan getCompleteScan();
 
 private:
+  std::shared_ptr<ControllerStateMachineImpl> state_machine_;
+  // TODO should be dedicated ControlMsgDecoder
+  MsgDecoder control_msg_decoder_;
+  std::shared_ptr<UdpClientImpl> control_udp_client_;
+  // TODO should be dedicated DataMsgDecoder
```

Comment:
is this to be done in the next pbi?

Reply:
This is to be done when we start differentiating between control messages and data messages.